thamos/config.py

```diff
@@ -163,10 +164,17 @@ def __init__(self):
     def api_url(self):
         """Get URL to Thoth's API."""
         if not self._api_url:
-            self._api_url = self.api_discovery(self.content["host"])
+            self._api_url = self.api_discovery(self.content["host"])[0]
 
         return self._api_url
 
+    @property
+    def thoth_search_ui_url(self):
+        if not self._thoth_search_ui_url:
+            self._thoth_search_ui_url = self.api_discovery(self.content["host"])[1]
```

Comment:
To reduce connections, would it make sense to refactor `api_discovery` and assign `_thoth_search_ui_url` `_api_url` directly there? It could also keep `api_discovery` backwards compatible with respect to the return value.

Reply:
Thanks, I changed the implementation in https://github.com/thoth-station/thamos/pull/1083/commits/e783e70059c3a62304967ad239a785683d6d2116 💯